Rules/Strings.resx

```diff
@@ -1,996 +1,1023 @@
-  </data>
-  <data name="AvoidAssignmentToReadOnlyAutomaticVariableCommonName" xml:space="preserve">
-    <value>Changing automtic variables might have undesired side effects</value>
-  </data>
-  <data name="AvoidAssignmentToReadOnlyAutomaticVariableDescription" xml:space="preserve">
-    <value>This automatic variables is built into PowerShell and readonly.</value>
-  </data>
-  <data name="AvoidAssignmentToReadOnlyAutomaticVariableError" xml:space="preserve">
-    <value>The Variable '{0}' cannot be assigned since it is a readonly automatic variable that is built into PowerShell, please use a different name.</value>
-  </data>
-  <data name="AvoidAssignmentToAutomaticVariableName" xml:space="preserve">
-    <value>AvoidAssignmentToAutomaticVariable</value>
-  </data>
-  <data name="AvoidAssignmentToReadOnlyAutomaticVariableIntroducedInPowerShell6_0Error" xml:space="preserve">
-    <value>Starting from PowerShell 6.0, the Variable '{0}' cannot be assigned any more since it is a readonly automatic variable that is built into PowerShell, please use a different name.</value>
-  </data>
-  <data name="AvoidUsingCmdletAliasesMissingGetPrefixError" xml:space="preserve">
-    <value>'{0}' is implicitly aliasing '{1}' because it is missing the 'Get-' prefix. This can introduce possible problems and make scripts hard to maintain. Please consider changing command to its full name.</value>
-  </data>
-  <data name="PossibleIncorrectUsageOfAssignmentOperatorDescription" xml:space="preserve">
-    <value>'=' or '==' are not comparison operators in the PowerShell language and rarely needed inside conditional statements.</value>
-  </data>
-  <data name="PossibleIncorrectUsageOfAssignmentOperatorError" xml:space="preserve">
-    <value>Did you mean to use the assignment operator '='? The equality operator in PowerShell is 'eq'.</value>
-  </data>
-  <data name="PossibleIncorrectUsageOfRedirectionOperatorCommonName" xml:space="preserve">
-    <value>'&gt;' is not a comparison operator. Use  '-gt' (greater than) or '-ge' (greater or equal).</value>
-  </data>
-  <data name="PossibleIncorrectUsageOfRedirectionOperatorDescription" xml:space="preserve">
-    <value>When switching between different languages it is easy to forget that '&gt;' does not mean 'great than' in PowerShell.</value>
-  </data>
-  <data name="PossibleIncorrectUsageOfRedirectionOperatorError" xml:space="preserve">
-    <value>Did you mean to use the redirection operator '&gt;'? The comparison operators in PowerShell are '-gt' (greater than) or '-ge' (greater or equal).</value>
-  </data>
-  <data name="PossibleIncorrectUsageOfRedirectionOperatorName" xml:space="preserve">
-    <value>PossibleIncorrectUsageOfRedirectionOperator</value>
-  </data>
-  <data name="PossibleIncorrectComparisonWithNullSuggesteCorrectionDescription" xml:space="preserve">
-    <value>Use $null on the left hand side for safe comparison with $null.</value>
-  </data>
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Not sure why the changes here are so big. Can easily redo them to make them small if desired. The VS magic here does things I don't really want to know about, but I can play with it.